packages/clerk-js/src/ui/signUp/util.ts

```diff
@@ -0,0 +1,27 @@
+import { SignUpResource } from '@clerk/types';
+
+type CompleteSignUpFlowProps = {
+  signUp: SignUpResource;
+  signUpUrl?: string;
+  navigate: (to: string) => Promise<void>;
+  handleComplete?: () => Promise<void>;
+};
+
+export const CompleteSignUpFlow = ({
```

Comment:
🙃 Since this is a utility function you can just name it `completeSignUpFlow`. Components are usually titleized.